examples/pytorch/eeg-gcnn/README.md

```diff
@@ -0,0 +1,55 @@
+# DGL Implementation of EEG-GCNN Paper
+This example is a simplified version that presents how to utilize the original EEG-GCNN model proposed in the paper [EEG-GCNN](http://proceedings.mlr.press/v136/wagh20a.html), implemented with DGL library. The original code is [here](https://github.com/neerajwagh/eeg-gcnn).
```

Comment:
Is it possible to provide a brief summary about the discrepancies between the original version and the simplified version?

Reply:
There is a note under performance section. Do you want me to move it to the top?

Reply:
I see. It looks good to me now.